pyecharts/echarts/axis.py

```diff
@@ -29,7 +29,10 @@ def __init__(self, **kwargs):
 
 class YAxisLabel(AxisLabel):
     def __init__(self, formatter=None, **kwargs):
-        if not isinstance(formatter, types.FunctionType):
+        if (
```

Comment:
YAxisLabel could have one small unit test.

Reply:
I had added unit tests for `YAxisLabel` to the `test/test_axis.py` file.